api_tests/base/test_sloan_study.py

```diff
@@ -50,10 +51,11 @@ def providers(self, user):
     def flags(self, user):
         Flag.objects.filter(name__in=SLOAN_FLAGS, percent=50).update(everyone=None)
 
+    @retry_assertion()
```

Comment:
Because these aren't mocked any more we have a 1% chance of failure (you can't send it to 100% or 0%) retry assert ensures the odds of a prob fail are .01^3